packages/app/src/app/pages/NewDashboard/Content/routes/Repositories/useFilteredItems.ts

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect } from 'react';
+import { compareDesc } from 'date-fns';
+import { useOvermind } from 'app/overmind';
+import { DashboardGridItem } from 'app/pages/NewDashboard/types';
+
+type Params = {
+  path?: string;
+};
+
+export const useFilteredItems = (params: Params) => {
+  const param = params.path || '';
+  const {
+    state: {
+      dashboard: {
+        getFilteredSandboxes,
+        sandboxes,
+        filters,
+        orderBy: sandboxesOrder,
+      },
+    },
+  } = useOvermind();
+  const [items, setItems] = useState<Array<DashboardGridItem>>([]);
+  const sandboxesForPath =
+    sandboxes.REPOS && param
+      ? getFilteredSandboxes(sandboxes.REPOS[param].sandboxes || [])
+      : [];
+
+  const repos = (sandboxes.REPOS && Object.values(sandboxes.REPOS)) || [];
```

Comment:
Same here, I think it's better to not use `Object.values` but rather something like `== null`.

Reply:
fixed

Reply:
Ohhh, no because I actually want the values of each key in the object